x/superfluid/README.md

```diff
@@ -1048,3 +1072,72 @@ the `IntermediaryAccount` will be slashed by less than the
 TODO - expand on this Uses `lockup` accumulator to find total amount of
 synthetic locks for a given `IntermediaryAccount` (Superfluid Asset +
 Validator pair)
+
+## Pool tokens vs non-pool Tokens
+
+The superfluid module treats pool tokens and non-pool tokens differently
+when calculating the staking power of a denom. This is because the pool tokens
+actually represent an amount of the underlying osmo in the pool, while non-pool 
+tokens just have an equivalent value that is determined by the twap price of the 
+token.
+
+### Adding a new non-pool token
+
+To add a new non-pool token to the superfluid module, a proposal needs to be sent
+specifying the new token's denom, the asset type, and a price route. 
+
+The price route needs to start in a pool containing the token and end in a pool
+containing osmo. 
+
+Example:
+
+```go
+types.SuperfluidAsset{
+	Denom: btcDenom, 
+	AssetType: types.SuperfluidAssetTypeNative, 
+	PriceRoute: []*poolmanagertypes.SwapAmountInRoute{{PoolId: poolId, TokenOutDenom: bondDenom}}
+}
+```
+
+### Delegating
+
+Once the new token is added, users can delegate to the new token in the same way
+as they would with a pool token. The module will calculate the staking power of the
+token based on the twap price of the token.
+
+Example:
+
+```go
+	types.MsgLockAndSuperfluidDelegate{
+		Sender:  userAddr.String(),
+		Coins:   sdk.NewCoins(sdk.NewCoin(btcDenom, btcStakeAmount)),
+		ValAddr: validator.GetOperator(),
+	}
+```
+
+### Querying delegations
+
+To check the superfluid delegations for a user the following query can be used:
+
+```go
+types.SuperfluidDelegationsByDelegatorRequest{DelegatorAddress: userAddr.String()}
+```
+
+### Rewards
```

Comment:
Is there anything about limits for how much of osmosis stake can be comprised of the native denom? (or rather, the max percent of rewards that can go to the native token). I think this isn't implemented yet though, just came to mind as I was reading this.

Reply:
Left a comment in https://github.com/osmosis-labs/osmosis/pull/8335

We can just add to the docs with that PR